src/modules/commander/HealthAndArmingChecks/checks/rcAndDataLinkCheck.cpp

```diff
@@ -51,7 +51,7 @@ void RcAndDataLinkChecks::checkAndReport(const Context &context, Report &reporte
 
 		if (!reporter.failsafeFlags().manual_control_signal_lost && _last_valid_manual_control_setpoint > 0) {
 
-			events::send(events::ID("commander_rc_lost"), {events::Log::Critical, events::LogInternal::Info},
+			events::send(events::ID("commander_rc_lost"), {events::Log::Info, events::LogInternal::Info},
```

Comment:
we already have the failsafe action that tells the user that the manual control has been lost, firing also this message as critical just increases entropy with no new information

Reply:
ok after testing it myself I understand what you mean:
The critical messages cause a yellow pop-up because it's critical to know immediately.
In the case of flying with stick input available and then losing it there are in total 3 critical messages and you mainly see the first one as yellow pop-up which is not helpful in this case:
<img width="812" alt="Screenshot_1" src="https://github.com/PX4/PX4-Autopilot/assets/4668506/f165ea9a-afd6-4a95-bc63-25c74b5e13a6">
<img width="328" alt="Screenshot_2" src="https://github.com/PX4/PX4-Autopilot/assets/4668506/9c6bd9af-189c-4788-ad91-03e116975c4c">
I suggest we set it to warning instead of info such that the message is still red instead of just white in the message stack but doesn't cause the pop-up:
<img width="822" alt="Screenshot_3" src="https://github.com/PX4/PX4-Autopilot/assets/4668506/aa3291a3-8246-4f0d-8d09-2633c71437ef">
![image](https://github.com/PX4/PX4-Autopilot/assets/4668506/377159f6-b29d-4d19-941f-c8f2686656d4)

Reply:
I put that in b9c388f9b1929b6495ebb32bfb670c7b59e31b4e

Reply:
Doesn't show the right way in all implementations 🤦‍♂️ I'll take it out and bring back once that's fixed.